end-to-end-tests/specs/screenshot.spec.js

```diff
@@ -82,18 +82,28 @@ function runResultsTests(){
 
 }
 
+function runIndependentResultsTests() {
+    describe("oncoprint tests", function() {
+        it("shows promoter mutations", function() {
+            goToUrlAndSetLocalStorage(`${CBIOPORTAL_URL}/index.do?cancer_study_id=hcc_inserm_fr_2015&Z_SCORE_THRESHOLD=2&RPPA_SCORE_THRESHOLD=2&data_priority=0&case_set_id=hcc_inserm_fr_2015_sequenced&gene_list=AKR1C1%2520AKR1C2%2520AKR1C4%2520CYP3A4%2520CYP3A43%2520CYP3A5%2520CYP3A7%2520UGT2B15%2520UGT2B17%2520UGT2B7%2520TERT&geneset_list=+&tab_index=tab_visualize&Action=Submit&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=hcc_inserm_fr_2015_mutations`);
+            waitForOncoprint(10000);
+            var res = browser.checkElement('#oncoprint');
+            assertScreenShotMatch(res);
+        });
+    });
+}
+
 describe('result page screenshot tests', function(){
     before(function(){
         var url = `${CBIOPORTAL_URL}/index.do?tab_index=tab_visualize&cancer_study_list=coadread_tcga_pub&cancer_study_id=coadread_tcga_pub&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=coadread_tcga_pub_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=coadread_tcga_pub_gistic&Z_SCORE_THRESHOLD=2.0&case_set_id=coadread_tcga_pub_nonhypermut&case_ids=&gene_list=KRAS+NRAS+BRAF&gene_set_choice=user-defined-list&Action=Submit&show_samples=false&`;
         goToUrlAndSetLocalStorage(url);
     });
 
-    runResultsTests()
-
+    runResultsTestSuite()
```

Comment:
what is reason for wrapping these?

Reply:
i see, to test both session and url based, yes?

Reply:
i have tests that i want to be able to run in the session mode and url parameters mode. and they need to be separate because what is now called `runResultsTests` runs in a suite, the tests are not independent

Reply:
right, yeah I didnt see your second comment before commenting

Reply:
wait but I just realized my test doesnt even do that so this doesnt make sense..